src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1546,6 +1557,56 @@ protected override void HandleThreadsRequestAsync(IRequestResponder<ThreadsArgum
             responder.SetResponse(response);
         }
 
+        private static long FileTimeToPosix(FILETIME ft)
```

Comment:
nit: This is in the `DebugAdapterBase` region. Move to `Utility` region (Line 266)